src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}

```

Comment:
```suggestion
.stars {
}
```

redundant, rewrite all stars to span

Reply:
Thanks, resolved